ccf-dynamodb/README.md

```diff
@@ -0,0 +1,53 @@
+# Copyright Cleared Feed DynamoDB Provisioner
+
+Provisions DynamoDB table that will be used to store client settings for Copyright Cleared Feed.
+
+## Provisioning new CCF DynamoDB table
+
+```sh
+cd ccf-dynamodb
+
+# Build the docker image
+docker build -t ccf-dynamodb:local .
+
+# Fill in the AWS credentials and region e.g. N. Virginia
```

Comment:
It will be nice to mention which IAM user to be used to generate the necessary credentials. If we don't have a user with the necessary policies in order the scripts to be executed, we should create one.

Reply:
The necessary policies will be added to the `upp-provisioner` user, I will add it to the comment.